osf/models/preprint_provider.py

```diff
@@ -6,12 +6,22 @@
 from osf.models.base import BaseModel, ObjectIDMixin
 from osf.models.licenses import NodeLicense
 from osf.models.subject import Subject
+from osf.models.user import Email, SocialAccount
 from osf.utils.datetime_aware_jsonfield import DateTimeAwareJSONField
 from osf.utils.fields import EncryptedTextField
 
 from website.util import api_v2_url
 
 
+class PreprintProviderLink(ObjectIDMixin, BaseModel):
```

Comment:
:+1: for making a generic model and naming the preprint field that references it something specific to preprints.